harold/plugins/deploy.py

```diff
@@ -170,8 +170,9 @@ class DeployHoldListener(DeployListener):
 
     def _handle_request(self, request):
         reason = request.args['reason'][0]
-        salon_name = request.args['salon_name'][0]
-        self.monitor.hold(self.monitor.irc, None, salon_name, reason)
+        salon_name = request.args['salon'][0]
```

Comment:
A few examples where this param is `salon` instead of `salon_name`:
https://github.com/spladug/harold/blob/f7f291f7fbef5d8ba59aa2b16f5d734ced4b96d3/harold/plugins/deploy.py#L158

https://github.com/spladug/harold/blob/f7f291f7fbef5d8ba59aa2b16f5d734ced4b96d3/harold/plugins/deploy.py#L148